lightify/__init__.py

```diff
@@ -962,6 +966,21 @@ def _next_seq(self):
             self.__seq = (self.__seq + 1) % 256
             return self.__seq
 
+    def set_loglevel(self, debugLevel):
+        self.__logger.setLevel(debugLevel)
+        level_str = 'NOTSET'
+        if debugLevel == logging.DEBUG:
+            level_str = 'DEBUG'
+        elif debugLevel == logging.INFO:
+            level_str = 'INFO'
+        elif debugLevel == logging.WARNING:
+            level_str = 'WARNING'
+        elif debugLevel == logging.ERROR:
+            level_str = 'ERROR'
+        elif debugLevel == logging.CRITICAL:
+            level_str = 'CRITICAL'
+        self.__logger.info("set_loglevel to '%s'", level_str)
```

Comment:
I believe you can get rid of `level_str` and just use `logging.getLevelName(level)`.
```suggestion
        self.__logger.info('set log level to %s', logging.getLevelName(level))
```

Reply:
I looked for a helper function for outputting the level name - sorry I'm a bit of a python newbie . Hopefully this function isn't for python 3.x

Reply:
No, it should work in Python 2.7 as well: https://docs.python.org/2/library/logging.html.